CGAL_ImageIO/src/CGAL_ImageIO/CMakeLists.txt

```diff
@@ -42,36 +42,40 @@ if(NOT CGAL_HEADER_ONLY)
 
     use_essential_libs()
 
-    include_directories( SYSTEM ${CGAL_3RD_PARTY_INCLUDE_DIRS} ${CGAL_ImageIO_3RD_PARTY_INCLUDE_DIRS} .)
-
-    link_directories    ( ${CGAL_LIBRARIES_DIR} ${CGAL_3RD_PARTY_LIBRARIES_DIRS} )
   endif(OPENGL_FOUND)
 
-else (NOT CGAL_HEADER_ONLY)
-  cache_set(CGAL_ImageIO_3RD_PARTY_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR})
+if (CGAL_HEADER_ONLY)
+  cache_set(CGAL_ImageIO_3RD_PARTY_INCLUDE_DIRS ${CGAL_ImageIO_3RD_PARTY_INCLUDE_DIRS} ${CMAKE_CURRENT_SOURCE_DIR})
 endif()
```

Comment:
No, because this is when we are in header only that we must add the directory CGAL_ImageIO/src/CGAL_ImageIO/ to the list of CGAL_ImageIO_3RD_PARTY_INCLUDE_DIRS.

When we are not in header only, this is not needed.